AjobK/Athena
# Node Update #closes 27

# Description
Updated Athena based on Node Version 10.14.2
**MAKE SURE TO UPDATE NODE TO LTS VERSION!**
[Node Download](https://nodejs.org/en/)

## Changes
- Updated package.json
  - Updated package-lock.json

### Hot to test
- Open branch `node-update`
  - Check if `npm run dev` still works
  - Approve this PR
